suppress.xml

```diff
@@ -1,24 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
-    <suppress>
+    <suppress until="2024-09-01Z">
         <notes><![CDATA[
-          File name: structured-logging-1.9.14.jar
-          ]]></notes>
-        <packageUrl regex="true">^pkg:maven/uk\.gov\.companieshouse/structured\-logging@.*$</packageUrl>
-        <cpe>cpe:/a:apache:log4j</cpe>
-    </suppress>
-
-    <suppress until="2023-05-25Z">
-        <notes><![CDATA[
-           File name: spring-web-5.3.22.jar
-
-           Suppressed for 6 months
-
-           A JIRA ticket has been created to track this vulnerability - https://companieshouse.atlassian.net/browse/DEBT-1742
-
-           It is unlikely that this library will be updated and therefore that the suppression will be removed.
-           ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.springframework/spring\-web@.*$</packageUrl>
-        <cve>CVE-2016-1000027</cve>
+   file name: jackson-databind-2.15.2.jar
```

Comment:
A more complete note here would be useful.

Reply:
What more should be added? what is added in the suppression is fairly standard

Reply:
A bit of info about what the vulnerability is and why it doesn't need to be fixed or similar I guess would be handy. Just something so the next team who pick up have more to go on than a CVE value

Reply:
done